src/styling-panel-definition.js

```diff
@@ -52,14 +52,17 @@ export const getStyleEditorDefinition = () => ({
                   type: 'boolean',
                   translation: 'properties.fontColor',
                   component: 'dropdown',
+                  defaultValue: false,
```

Comment:
I think we made a mistake here in the past. A dropdown and boolean should not be together

Reply:
Ah I see I forgot to remove it from here, I did in the later PRs. I'll fix it here.